test/e2e/fixture-builder.js

```diff
@@ -479,10 +479,7 @@ class FixtureBuilder {
               caveats: [
                 {
                   type: 'restrictReturnedAccounts',
-                  value: [
```

Comment:
I have attempted to extract this here: https://github.com/MetaMask/metamask-extension/pull/29783